src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,69 @@
+import lotto.model.dto.LottoResultDto;
+import lotto.model.dto.LottoTicketDto;
+import lotto.view.Input;
+import lotto.view.Output;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoController {
+
+    public static void play() {
+        PurchaseAmount purchaseAmount = new PurchaseAmount(Input.getPurchaseAmount());
+        validatePurchaseAmount(purchaseAmount);
+        int manualPurchaseCount = Input.getManualPurchaseCount();
+        validateMnaualPurchaseCount(purchaseAmount, manualPurchaseCount);
+        LottoTickets lottoTickets = buyLottoTickets(purchaseAmount, manualPurchaseCount);
+
+        Output.printPurchaseCount(manualPurchaseCount, lottoTickets.getSize() - manualPurchaseCount);
+        Output.printLottoTickets(parseDto(lottoTickets));
+
+        LottoTicket winningTicket = parseLottoTicket(Input.getWinningTicket());
+        LottoNumber bonusNumber = new LottoNumber(Input.getBonusNumber());
+        WinningLottoTicket winningLottoTicket = new WinningLottoTicket(winningTicket, bonusNumber);
+
+        LottoResult lottoResult = lottoTickets.makeWinningResult(winningLottoTicket);
+        Output.printResult(new LottoResultDto(lottoResult.calculateReturnRate(purchaseAmount), lottoResult.makeLottoResultMap()));
+    }
+
+    private static void validatePurchaseAmount(PurchaseAmount purchaseAmount) {
+        if (purchaseAmount.getPurchaseAmount() < LottoTicket.PRICE) {
+            throw new IllegalArgumentException("구매 금액은 최소 1,000원 이상이어야 합니다.");
+        }
+    }
+
+    private static void validateMnaualPurchaseCount(PurchaseAmount purchaseAmount, int manualPurchaseCount) {
+        if (purchaseAmount.getPurchaseAmount() < (long) manualPurchaseCount * LottoTicket.PRICE) {
+            throw new IllegalArgumentException("수동으로 구매할 로또 수가 구매 금액보다 많습니다.");
+        }
+    }
+
+    private static LottoTicket parseLottoTicket(String lottoTicket) {
+        return Arrays.stream(lottoTicket.split(", "))
+                .map(Integer::parseInt)
+                .map(LottoNumber::new)
+                .collect(Collectors.collectingAndThen(Collectors.toList(), LottoTicket::new));
+    }
+
+    private static List<LottoTicketDto> parseDto(LottoTickets lottoTickets) {
+        return lottoTickets.getLottoTickets()
+                .stream()
+                .map(LottoTicketDto::new)
+                .collect(Collectors.toList());
+    }
+
+    public static LottoTickets buyLottoTickets(PurchaseAmount purchaseAmount, int manualCount) {
+        List<LottoTicket> lottoTicketList = Input.getManualLottoTickets(manualCount).stream()
+                .map(LottoController::parseLottoTicket)
+                .collect(Collectors.toList());
+
+        purchaseAmount.spend((long) manualCount * LottoTicket.PRICE);
+        LottoTickets lottoTickets = new LottoTickets(lottoTicketList);
+        lottoTickets.add(LottoTicketGenerator.generate(purchaseAmount));
+        return lottoTickets;
```

Comment:
`LottoTicketGenerator.generate(purchaseAmount)`의 결과도 `LottoTickets` 타입으로 결과를 반환하고 
LottoTickets.`add`의 결과 값을 새로운 (합치는 두 LottoTickets의 리스트들을 합친) `LottoTickets` 객체로 만들어보면 어떨까요? 🙂


추가적으로 `manualLottoTickets`, `autoLottoTickets`와 같이 네이밍을 더 명확하게해주면 코드만봐도 `lottoTickets`가 어떻게 구성되어있는지가 쉽게 파악될 것 같아요. 😉

```suggestion
        purchaseAmount.spend((long) manualCount * LottoTicket.PRICE);
        LottoTickets manualLottoTickets = Input.getManualLottoTickets(manualCount).stream()
                .map(LottoController::parseLottoTicket)
                .collect(Collectors.collectingAndThen(Collectors.toList(), LottoTickets::new));

        LottoTickets autoLottoTickets = LottoTicketGenerator.generate(purchaseAmount);
        return manualLottoTickets.add(autoLottoTickets);
```

Reply:
반영하였습니다.